assimilation_code/programs/obs_utils/obs_info.f90

```diff
@@ -222,11 +228,24 @@ program obs_info
                                              trim(get_name_for_type_of_obs(i))//", ", &
                                              oinfo(i)%count, ", ", trim(mid_string)
       else
-         write(ounit, '(A,I8)') get_name_for_type_of_obs(i), oinfo(i)%count
+         write(ounit, '(A32,I8)') get_name_for_type_of_obs(i), oinfo(i)%count
```

Comment:
why did you specify a fixed width?  get_name_for_type_of_obs() returns a length 31 string, so this seems unnecessary.

Reply:
i wanted to use identical formats for both regular obs and identity obs.